scripts/test-acceptance.sh

```diff
@@ -19,6 +19,7 @@
 cd `dirname $BASH_SOURCE` && cd ..
 
 # Note: support BASE_URL as a synonym for ORA_SANDBOX_URL for backward compatibility
+ORA_SANDBOX_URL = 'staff:edx@mushiora.sandbox.edx.org -o StrictHostKeyChecking=no'
```

Comment:
Yes, you are right. I was testing this on the jenkins run. I would revert this file before merging :)